.kilocode/skills/add-wshcmd/SKILL.md

```
@@ -0,0 +1,921 @@
+
+# With block targeting
+wsh mycommand -b 2 arg1
+
+# Complex example
+wsh mycommand -v --name "example" arg1 arg2
+```
+
+Additional notes, tips, or warnings about the command.
+
+---
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)